R/util.R

```diff
@@ -2417,6 +2417,47 @@ week <- function(date, unit="year") {
   }
 }
 
+
+#' API to calculate duration between the start_date and the end_date in the provided time unit.
+time_between <- function(start_date, end_date=lubridate::today(), unit = "years") {
```

Comment:
end_date default here looks unnecessary for this intermediate function.

Reply:
Removed default for end_date argument.